docs/examples/writing_drivers/Creating-Instrument-Drivers.ipynb

```diff
@@ -98,25 +153,187 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Logging\n",
-    "Every QCoDeS module should have its own logger that is named with the name of the module. So to create a logger put a line at the top of the module like this:\n",
-    "```\n",
-    "log = logging.getLogger(__name__)\n",
-    "```\n",
-    "Use this logger only to log messages that are not originating from an `Instrument` instance. For messages from within an instrument instance use the `log` member of the `Instrument` class, e.g\n",
-    "```\n",
-    "self.log.info(f\"Could not connect at {address}\")\n",
-    "```\n",
-    "This way the instrument name will be prepended to the log message and the log messages can be filtered according to the instrument they originate from. See the example notebook of the logger module for more info ([offline](../logging/logging_example.ipynb),[online](https://nbviewer.jupyter.org/github/QCoDeS/Qcodes/tree/master/docs/examples/logging/logging_example.ipynb)).\n",
+    "## Simple driver examples (Decorator style)"
```

Comment:
i'd suggest to move the "decorator style" to below the "manual instruments" section. this would be inline with the statement that the "conventional style" is still the default and not going anywhere. also note that this example notebook can't really talk about "details of the ways of adding parameters" before a reader has seen an example of a driver in the "visa isntrument" section where many details of how it all works together are explained (mostly in the comments in the code). would you agree?

Reply:
Agreed, "method style" and "decorator style"